### PR TITLE
QOLDEV-414 Handling private links in any sensitive page

### PR DESCRIPTION
**Pull request by ghazal-salehi**
Any page that has sensitive data has a Quick Exit Component. This is a good indication to identify if the links in a page need to be kept unchanged when visited.

https://oss-uat.clients.squiz.net/community/getting-support-health-social-issue/support-victims-abuse/domestic-family-violence/find-local-support
https://oss-uat.clients.squiz.net/community/getting-support-health-social-issue/support-victims-abuse/reporting-abuse
https://oss-uat.clients.squiz.net/community/women/support-for-women
https://oss-uat.clients.squiz.net/community/getting-support-health-social-issue/support-victims-abuse/sexual-abuse-assault/sexual-abuse-assault-getting-help
[About domestic and family violence and coercive control](https://oss-uat.clients.squiz.net/community/getting-support-health-social-issue/support-victims-abuse/domestic-family-violence/what-is-domestic-and-family-violence/what-are-the-patterns-of-domestic-violence/about-domestic-and-family-violence-and-coercive-control) 
[What are the patterns of violence or abuse?](https://oss-uat.clients.squiz.net/community/getting-support-health-social-issue/support-victims-abuse/domestic-family-violence/what-is-domestic-and-family-violence/what-are-the-patterns-of-domestic-violence)

### CODE REVIEW

File: src/assets/_project/_blocks/qg-main.js
Comment (RossWebsterWork) [inline src/assets/_project/_blocks/qg-main.js:1]: I'm not too familiar with the SWE conventions, but would the "private-content" class need to start with "qg-"?
And I think this wouldn't apply the stylings if JavaScript on the page is disabled, is that something that could be an issue or not really?

**Comment by RossWebsterWork** [comment]
Also, while not completely related to these changes, the "Close" button in the "Leave your feedback" section of the footer is completely white, I'm not sure if its related to the error with the failing footer-feedback test or not.
image

File: src/assets/_project/_blocks/qg-main.js
Comment (ThrawnCA) [reply src/assets/_project/_blocks/qg-main.js:1]: If JavaScript is disabled, then Quick Exit is busted anyway.

**Review by ThrawnCA — APPROVED** [review]

**Review by RossWebsterWork — APPROVED** [review]
The footer-feedback alert also has white text but isn't directly related to this change.
Otherwise it all seems good.
image

**Pull request merged**